src/kili/llm/presentation/client/llm.py

```diff
@@ -84,10 +100,76 @@ def export(
             warnings.warn(str(excp), stacklevel=2)
             return None
 
+    def list_models(self, organization_id: str, fields: Optional[List[str]] = None):
+        """List models of given organization."""
+        converted_filters = OrganizationModelFilters(
+            organization_id=organization_id,
+        )
+
+        return list(
+            self.kili_api_gateway.list_models(
+                filters=converted_filters,
+                fields=fields if fields else DEFAULT_ORGANIZATION_MODEL_FIELDS,
+            )
+        )
+
+    def get_model(self, model_id: str, fields: Optional[List[str]] = None):
+        return self.kili_api_gateway.get_model(
+            model_id=model_id,
+            fields=fields if fields else DEFAULT_ORGANIZATION_MODEL_FIELDS,
+        )
+
+    def create_model(self, organization_id: str, model: dict):
+        credentials_data = model["credentials"]
+        model_type = ModelType(model["type"])
+
+        if model_type == ModelType.AZURE_OPEN_AI:
+            credentials = AzureOpenAICredentials(**credentials_data)
+        elif model_type == ModelType.OPEN_AI_SDK:
+            credentials = OpenAISDKCredentials(**credentials_data)
+        else:
+            raise ValueError(f"Unsupported model type: {model['type']}")
+
+        model_input = ModelToCreateInput(
+            credentials=credentials,
+            name=model["name"],
+            type=model_type,
+            organization_id=organization_id,
+        )
+        return self.kili_api_gateway.create_model(model=model_input)
+
+    def update_model(self, model_id: str, model: dict):
```

Comment:
Not sure here if we should rename it to `update_properties_in_model`, as the parameter as quite different to an `update_properties_in_assets`